### ISSUE TITLE
`--env-file` flag on Windows does not support parent file paths (`..\`)

### ISSUE THREAD

**Issue by kaminskypavel** [oven-sh/bun#9813]
### What version of Bun is running?

1.1.0+5903a6141

### What platform is your computer?

bundows

### What steps can reproduce the bug?

`bun --env-file=../../.env test`

### What is the expected behavior?

Bun to load the .env file and indicate it in the console

### What do you see instead?

nothing

**Comment by is-jonreeves** [comment]
The `--env-file` option works ok for me in Windows 10 on both powershell and git-bash, (even when specified multiple times).

I hadn't tried out relative `../../` paths "above" the current one before, but I just gave it a shot and seems ok. Only other thing of note is that I have a `bunfig.toml` file with a `root` option set on the `[test]` section:

`/bunfig.toml`
```toml
...
[test]
root = "./src"
preload = ["./test/setup.ts"]
coverage = false
coverageThreshold = { line = 0.7, function = 0.8, statement = 0.9 }
```

Maybe that will help you? If not double check the current directory you are running the command from.

From the folder with my package.json, these commands work fine for me...
- `bun test`
- `bun --env-file=.env --env-file=.env.local --env-file=.env.development --env-file=.env.develop.local test`
- `bun --env-file=../../.env.a --env-file=../../.env.b test`

### Other Observations:
Likely not related, but I did notice that with `bun test` if you set `NODE_ENV` as you execute the command, while the value does get passed in, it doesn't appear to influence which dotenvs to load, unlike `bun build` and `bun run`. Maybe that's intended, I'm unsure. Its just odd that it doesn't behave the same.

**Comment by kaminskypavel** [comment]
@is-jonreeves thanks for your detailed input.

I've tested your assumption and its definitely a bug.
`bun test` does load a `.env` but ONLY if it's located in the same directory. 

so in the following scenario 
```json
    "test": "bun --env-file=././../.env test",
    "dev": "bun --env-file=./../../.env --hot src/index.ts"
```

`bun:dev` ✅ works 
`bun:test` ❌ fails

**Comment by gvilums** [comment]
> @is-jonreeves thanks for your detailed input.
> 
> I've tested your assumption and its definitely a bug. `bun test` does load a `.env` but ONLY if it's located in the same directory.
> 
> so in the following scenario
> 
> ```json
>     "test": "bun --env-file=././../.env test",
>     "dev": "bun --env-file=./../../.env --hot src/index.ts"
> ```
> 
> `bun:dev` ✅ works `bun:test` ❌ fails

In your example scenario the paths are different: in the `test` command the path is `././../.env`, but in the `dev` command it's `./../../.env` (probably a typo?)

**Comment by kaminskypavel** [comment]
> > @is-jonreeves thanks for your detailed input.
> > I've tested your assumption and its definitely a bug. `bun test` does load a `.env` but ONLY if it's located in the same directory.
> > so in the following scenario
> > ```json
> >     "test": "bun --env-file=././../.env test",
> >     "dev": "bun --env-file=./../../.env --hot src/index.ts"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > `bun:dev` ✅ works `bun:test` ❌ fails
> 
> In your example scenario the paths are different: in the `test` command the path is `././../.env`, but in the `dev` command it's `./../../.env` (probably a typo?)

you are correct, this was a type, but even after fixing it , it had no effect on the outcome.
bun test just ignores the `-env-file=./../../.env`  path.

It does load the `.env` file if its located in the same folder as the `package.json`
